webapp/components/notifications/Notification/Notification.vue

```diff
@@ -2,20 +2,29 @@
   <ds-space :class="{ notification: true, read: notification.read }" margin-bottom="x-small">
     <no-ssr>
       <ds-space margin-bottom="x-small">
-        <hc-user :user="post.author" :date-time="post.createdAt" :trunc="35" />
+        <hc-user
+          :user="post.author || comment.author"
+          :date-time="post.createdAt || comment.createdAt"
+          :trunc="35"
+        />
       </ds-space>
       <ds-text color="soft">
-        {{ $t('notifications.menu.mentioned') }}
+        {{ $t('notifications.menu.mentioned', { resource: post.id ? 'post' : 'comment' }) }}
```

Comment:
```suggestion
        {{ $t('notifications.menu.mentioned', { resourceType }) }}
```

Reply:
I did and it gets refactored even more in the next PR 🍰 Send out notifications on CreateComment #1018